scripts/mod_loader/bootstrap/constants.lua

```diff
@@ -26,4 +26,9 @@ modApi.constants = {
   PILOT_CONFIG_POD_NORMAL = 1,
   PILOT_CONFIG_POD_ADVANCED = 2,
   PILOT_CONFIG_RECRUIT = 4,
+
+  SQUAD_INDEX_START = 0,
+  SQUAD_INDEX_END = 15,
+  SQUAD_INDEX_RANDOM = 8,
+  SQUAD_INDEX_CUSTOM = 9,
```

Comment:
Should probably use these constants in the altered squads code as I have a couple of bound checks there for the index